src/contracts/GPv2Settlement.sol

```diff
@@ -1,5 +1,6 @@
 // SPDX-License-Identifier: LGPL-3.0-or-newer
 pragma solidity ^0.7.5;
+pragma abicoder v2;
```

Comment:
If I remember correctly @josojo had concerns regarding using abicoder v2.

Reply:
This should not stick around when it comes to deployment time from what I remember.

Reply:
I believe this has something to do with passing structs along as function parameters? However it appears it is no longer in "experimental" phase so maybe this is alright.

Reply:
It's stable now and no longer experimental and used for calling the `GPv2AllowanceManager.transferIn` method. We can also implement encoding and decoding of the memory by hand.